src/completed-list/completed-list.component.tsx

```diff
@@ -153,28 +74,156 @@ const CompletedList: React.FC<CompletedListProps> = ({ fulfillerStatus }) => {
   ];
 
   const tableRows = useMemo(() => {
-    return paginatedWorkListEntries?.map((entry, index) => (
-      <CustomTableRow key={index} entry={entry} />
-    ));
-  }, [paginatedWorkListEntries]);
+    return paginatedWorkListEntries
+      ?.filter((item) => item.fulfillerStatus === fulfillerStatus)
+      .map((entry, index) => ({
+        ...entry,
+        id: entry.uuid,
+        date: {
+          content: (
+            <>
+              <span>{formatDate(parseDate(entry.dateActivated))}</span>
+            </>
+          ),
+        },
+        patient: {
+          content: (
+            <ConfigurableLink
+              to={`\${openmrsSpaBase}/patient/${entry.patient.uuid}/chart/laboratory-orders`}
+            >
+              {entry.patient.display.split("-")[1]}
+            </ConfigurableLink>
+          ),
+        },
+        orderNumber: { content: <span>{entry.orderNumber}</span> },
+        accessionNumber: { content: <span>{entry.accessionNumber}</span> },
+        test: { content: <span>{entry.concept.display}</span> },
+        action: { content: <span>{entry.action}</span> },
+        status: {
+          content: (
+            <>
+              <Tag>
+                <span
+                  className={styles.statusContainer}
+                  style={{ color: `${getStatusColor(entry.fulfillerStatus)}` }}
+                >
+                  <span>{entry.fulfillerStatus}</span>
+                </span>
+              </Tag>
+            </>
+          ),
+        },
+        orderer: { content: <span>{entry.orderer.display}</span> },
+        orderType: { content: <span>{entry.orderType.display}</span> },
+        urgency: { content: <span>{entry.urgency}</span> },
+      }));
+  }, [fulfillerStatus, paginatedWorkListEntries]);
 
   if (isLoading) {
     return <DataTableSkeleton role="progressbar" />;
   }
 
-  if (paginatedWorkListEntries?.length > 0) {
-    return <DataTable rows={tableRows} headers={tableColumns} useZebraStyles />;
-  } else {
+  if (paginatedWorkListEntries?.length >= 0) {
     return (
-      <div className={styles.tileContainer}>
-        <Tile className={styles.tile}>
-          <div className={styles.tileContent}>
-            <p className={styles.content}>
-              {t("noCompletedListToDisplay", "No Completed List to display")}
-            </p>
-          </div>
-        </Tile>
-      </div>
+      <DataTable rows={tableRows} headers={tableColumns} useZebraStyles>
+        {({
+          rows,
+          headers,
+          getHeaderProps,
+          getTableProps,
+          getRowProps,
+          onInputChange,
+        }) => (
+          <TableContainer className={styles.tableContainer}>
+            <TableToolbar
+              style={{
+                position: "static",
+                height: "3rem",
+                overflow: "visible",
+                backgroundColor: "color",
+              }}
```

Comment:
let's move this styling to the scss file

Reply:
the Toolbar doesn't really provide for `className` so thats the reason opted for `style={}`